[2023-02-20 01:14:50,318] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: autoetl-coordinates.get_metrics manual__2023-02-20T01:14:47.184136+00:00 [queued]>
[2023-02-20 01:14:50,331] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: autoetl-coordinates.get_metrics manual__2023-02-20T01:14:47.184136+00:00 [queued]>
[2023-02-20 01:14:50,332] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-02-20 01:14:50,334] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-02-20 01:14:50,335] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-02-20 01:14:50,351] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_metrics> on 2023-02-20 01:14:47.184136+00:00
[2023-02-20 01:14:50,361] {standard_task_runner.py:52} INFO - Started process 5453 to run task
[2023-02-20 01:14:50,365] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'autoetl-coordinates', 'get_metrics', 'manual__2023-02-20T01:14:47.184136+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/autoetl-coordinates.py', '--cfg-path', '/tmp/tmpy18fxrh_', '--error-file', '/tmp/tmpue4qz4gw']
[2023-02-20 01:14:50,368] {standard_task_runner.py:80} INFO - Job 24: Subtask get_metrics
[2023-02-20 01:14:50,440] {task_command.py:369} INFO - Running <TaskInstance: autoetl-coordinates.get_metrics manual__2023-02-20T01:14:47.184136+00:00 [running]> on host 83ee186d463f
[2023-02-20 01:14:50,519] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=autoetl-coordinates
AIRFLOW_CTX_TASK_ID=get_metrics
AIRFLOW_CTX_EXECUTION_DATE=2023-02-20T01:14:47.184136+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-20T01:14:47.184136+00:00
[2023-02-20 01:14:50,522] {logging_mixin.py:115} INFO - Load Data
[2023-02-20 01:14:51,285] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/operators/python.py:189 DtypeWarning: Columns (12,13,15,21) have mixed types.Specify dtype option on import or set low_memory=False.
[2023-02-20 01:14:51,331] {logging_mixin.py:115} INFO - Clean updated locations
[2023-02-20 01:14:51,990] {logging_mixin.py:115} WARNING - /opt/***/dags/autoetl-coordinates.py:184 FutureWarning: The default value of regex will change from True to False in a future version. In addition, single character regular expressions will *not* be treated as literal strings when regex=True.
[2023-02-20 01:14:52,070] {logging_mixin.py:115} WARNING - /opt/***/dags/autoetl-coordinates.py:185 FutureWarning: The default value of regex will change from True to False in a future version. In addition, single character regular expressions will *not* be treated as literal strings when regex=True.
[2023-02-20 01:14:52,281] {logging_mixin.py:115} INFO - Calculate Metrics
[2023-02-20 01:14:52,386] {logging_mixin.py:115} INFO - Create dataframe for metrics and save
[2023-02-20 01:14:52,418] {python.py:173} INFO - Done. Returned value was: None
[2023-02-20 01:14:52,434] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=autoetl-coordinates, task_id=get_metrics, execution_date=20230220T011447, start_date=20230220T011450, end_date=20230220T011452
[2023-02-20 01:14:52,514] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-20 01:14:52,559] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
